eo-maven-plugin/src/main/java/org/eolang/maven/tojos/ForeignTojo.java

```diff
@@ -404,4 +404,22 @@ public boolean equals(final Object other) {
     public int hashCode() {
         return Objects.hash(this.delegate);
     }
+
+    /**
+     * Return the attribute from the tojo.
+     * @param attribute The attribute from ForeignTojos.Attribute.
+     * @return The attribute.
+     */
+    private String attributeTojo(final ForeignTojos.Attribute attribute) {
+        final String path = this.delegate.get(attribute.key());
+        if (path == null) {
+            throw new IllegalStateException(
+                String.format(
+                    "There is no '%s' attribute in the tojo", attribute
```

Comment:
@Yanich96 I'm not sure if this string is formatted well because `attribute` does not seem to have `toString` method. Let's add a test at least for one attribute that would check exception message

Reply:
@maxonfjvipon done